interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is a way to impliment HTML in a JavaScript file, JSX uses camelCase for "className" instead of HTML's "class". Another syntax used in JSX is the empty fragments <></> used to enclose your components.
 
-Researched answer:
+Researched answer: JSX (JavaScript Syntax Extension) is an extension of JavaScript used in React, allowing you to impliment HTML within JavaScript. Instead of using the class attribute in HTML you would use className in JSX. For HTML an example would be <h1 class="example">Howdy!</h1> whereas in JSX you would use <h1 className="example">Howdy!</h1>
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-Your answer:
+Your answer: Yarn is a command used to create a React application, when running yarn it creates two extra iles package.json and yarn.lock
 
-Researched answer:
+Researched answer: Yarn (Yet Another Resource Negotiator) is a package manager created by Facebook. Yarn creates package.json in your React project directory as well as yarn.lock
 
 3. What is an anonymous function in JavaScript?
 
-Your answer:
+Your answer: An anonymous function is a function with no name that you still want to declare, it's created with ()
 
-Researched answer:
+Researched answer: Anonymous functions are functions without a name. You would often use them as arguments to other functions
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: Props (properties) are used in React to pull from different components you have stored in your React App. State values is how you access those props.
 
-Researched answer:
+Researched answer: Both props and state values are objects in JavaScript. Props are used to pass data from components and state values are controlled by the component itself.
 
 5. What is the DOM?
 
-Your answer:
+Your answer: The DOM is how you interact with the HTML elements inside of your React app
 
-Researched answer:
+Researched answer: DOM (Document Object Model) is the visual representation of code, it's the interface for HTML. 
```

Comment:
I think there is room to expand more on a lot of these answers for future assessments.